tests/support/server.tcl

```diff
@@ -182,6 +182,11 @@ proc tags_acceptable {err_return} {
         }
     }
 
+    if {$::external && [lsearch $::tags "external-ok"] == -1} {
```

Comment:
Using both makes it possible to opt-in a larger scope (e.g. `start_server` block) and opt-out a smaller scope (e.g. specific tests).

If you feel strongly about it, we can drop the `external-ok` and add the `external-skip` to all those tests that are not OK (roughly 50% of the files).

Reply:
In the other approach you can also opt out wither large blocks, or keep then in and op out smaller ones. 
I do feel stringy that it's a better approach. 

P. S. If you'll need to add the opt out to 50% of the files, then it means that with the current approach, you has to opt in 50% too (so same amount of lines). 
But even if it was 30/70 I still feel the opt out is the right way.
It's easier to say why something is incompatible rather than looking for things that aren't. And also, this way the CI will tell us when something needs to be excluded, rather than forgetting to include it.

Reply:
These are convincing arguments, `external-ok` is out now and `external:skip` is there to opt-out explicitly.